[2024-11-12T11:29:13.072+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-12T11:29:13.110+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: add_data_API.run_API_fin scheduled__2024-11-05T06:00:00+00:00 [queued]>
[2024-11-12T11:29:13.131+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: add_data_API.run_API_fin scheduled__2024-11-05T06:00:00+00:00 [queued]>
[2024-11-12T11:29:13.132+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-11-12T11:29:13.169+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): run_API_fin> on 2024-11-05 06:00:00+00:00
[2024-11-12T11:29:13.186+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=205) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-12T11:29:13.190+0000] {standard_task_runner.py:72} INFO - Started process 208 to run task
[2024-11-12T11:29:13.192+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'add_data_API', 'run_API_fin', 'scheduled__2024-11-05T06:00:00+00:00', '--job-id', '467', '--raw', '--subdir', 'DAGS_FOLDER/insert_data_fin.py', '--cfg-path', '/tmp/tmpscksfv5k']
[2024-11-12T11:29:13.194+0000] {standard_task_runner.py:105} INFO - Job 467: Subtask run_API_fin
[2024-11-12T11:29:13.311+0000] {task_command.py:467} INFO - Running <TaskInstance: add_data_API.run_API_fin scheduled__2024-11-05T06:00:00+00:00 [running]> on host 97974acde2c9
[2024-11-12T11:29:13.487+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='add_data_API' AIRFLOW_CTX_TASK_ID='run_API_fin' AIRFLOW_CTX_EXECUTION_DATE='2024-11-05T06:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-05T06:00:00+00:00'
[2024-11-12T11:29:13.490+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-11-12T11:29:13.492+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-11-12T11:29:13.494+0000] {logging_mixin.py:190} INFO - Current task name:run_API_fin state:running start_date:2024-11-12 11:29:13.111858+00:00
[2024-11-12T11:29:13.496+0000] {logging_mixin.py:190} INFO - Dag name:add_data_API and current dag run status:running
[2024-11-12T11:29:13.498+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-12T11:29:13.580+0000] {logging_mixin.py:190} INFO - Erreur lors de l'ex√©cution du script : Command '['python3', '/OneDrive/Documents/GitHub/Projet_EDD/script/getAPI_fin.py']' returned non-zero exit status 2.
[2024-11-12T11:29:13.582+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-12T11:29:13.601+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-12T11:29:13.602+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=add_data_API, task_id=run_API_fin, run_id=scheduled__2024-11-05T06:00:00+00:00, execution_date=20241105T060000, start_date=20241112T112913, end_date=20241112T112913
[2024-11-12T11:29:13.627+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-11-12T11:29:13.628+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-11-12T11:29:13.628+0000] {logging_mixin.py:190} INFO - Dag name:add_data_API queued_at:2024-11-12 11:29:09.191615+00:00
[2024-11-12T11:29:13.629+0000] {logging_mixin.py:190} INFO - Task hostname:97974acde2c9 operator:PythonOperator
[2024-11-12T11:29:13.691+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-12T11:29:13.709+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
